pkg/v1/features/tenant.go

```diff
@@ -0,0 +1,42 @@
+package features
```

Comment:
Added this here but I don't think we need to use it unless we make breaking SDK changes